node/p2p/routing.go

```diff
@@ -25,14 +25,11 @@ func ContentRouting() routing.ContentRouting {
 // Basically, this provides a way to discover peer addresses by respecting public keys.
 func PeerRouting(cfg Config) func(routingParams) (routing.PeerRouting, error) {
 	return func(params routingParams) (routing.PeerRouting, error) {
-		bpeers, err := cfg.bootstrapPeers()
-		if err != nil {
-			return nil, err
-		}
-
+		bpeers := nparams.BootstrappersInfos()
 		prefix := fmt.Sprintf("/celestia/%s", nparams.GetNetwork())
 		mode := dht.ModeAuto
 		if cfg.Bootstrapper {
+			bpeers = nil // no bootstrappers for a bootstrapper ¯\_(ツ)_/¯
```

Comment:
1. lol at the comment
2. Is it okay to `nil`ify `bpeers` here, instead of handling it earlier in the function? Maybe a nit but it seems a bit weird to set it _then_ clear it.

Reply:
True, Rene's suggestion should fix this

Reply:
I had another look at it after some time and made it even cleaner